docs/model-generation.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Generating models and classes using Modelina"
+weight: 200
+---
+
+Suppose you want to generate models for your application from the data types you have using the asyncapi generator. Then you should use Modelina instead of the asyncapi generator with a template.
```

Comment:
```suggestion
Use [AsyncAPI Modelina](/tools/modelina) to generate models for an application. Since models generation is something that you usually perform as an automated step through CI/CD, best way to use Modelina is through [AsyncAPI CLI](/tools/cli).

In case you work on a template for generating an application and part of template responsibilities is to provide models, you should integrate Modelina library instead of templating model structures using features from AsyncAPI Generator.
```